src/main/java/frc/robot/RobotContainer.java

```diff
@@ -71,4 +80,44 @@ public Command getAutonomousCommand() {
     return autoChooser.getSelected();
   }
 
+  // Ben. S, 2/4/2024 1:16AM
+  // Remove if unnecessary, or edit parts that aren't correct or lack needed info - I still tried my best 😭
+
+  // Shuffleboard is comprised of tabs, which then *can* contain widgets OR layouts made up of widgets
+  // 2020 code was confusing and too vague in the way they named their arguments, but I did my best to keep its structure
+  // Implement what you will
+  // 
+  // 
+  // Partial Shuffleboard hierarchy reference:
+  //
+  //| Smart dashboard
+  //|       Tab:                          Generally made up of layouts or their own independent widgets
+  //|          -> Widget:                 GUI for information, sometimes interactable
+  //|          -> Layout:                 They are essentially just subfolders for widgets used for organization
+  //|                   -> Widget
+  // 
+  // https://docs.wpilib.org/en/stable/docs/software/dashboards/shuffleboard/layouts-with-code/organizing-widgets.html#adding-widgets-to-layouts  
+  //  
+  // 2020 IndexerSubsystem.java: https://github.com/STMARobotics/frc-7028-2020/blob/b4bafc1792f9b4ddc187f8470b0808450ce86903/src/main/java/frc/robot/subsystems/IndexerSubsystem.java#L90C3-L98C4 
+  // 2020 RobotContainer.java:   https://github.com/STMARobotics/frc-7028-2020/blob/b4bafc1792f9b4ddc187f8470b0808450ce86903/src/main/java/frc/robot/RobotContainer.java#L240C3-L300C4 
```

Comment:
I don't think this comment belongs in the code. It doesn't explain anything that's specific to this codebase, the information is all part of the standard WPILib documentation.